conf/app.routes

```diff
@@ -2,7 +2,7 @@
 
 GET        /                @uk.gov.hmrc.testuser.controllers.TestUserControllerImpl.showCreateUserPage()
 
-POST       /                @uk.gov.hmrc.testuser.controllers.TestUserControllerImpl.createUser()
+POST       /user            @uk.gov.hmrc.testuser.controllers.TestUserControllerImpl.createUser()
```

Comment:
this is definitely better, is this currently live on / though rather than /user ? If so is there anything we need to do tell users of the change?